x-pack/plugins/triggers_actions_ui/public/application/sections/action_connector_form/action_form.tsx

```diff
@@ -433,9 +470,40 @@ export const ActionForm = ({
             );
           }
 
+          if (isSystemActionType) {
+            return (
+              <SystemActionTypeForm
+                actionItem={actionItem as RuleSystemAction}
+                actionConnector={actionConnector}
+                index={index}
+                key={`system-action-form-action-at-${actionItem.uuid}`}
+                setActionParamsProperty={setActionParamsProperty}
+                actionTypesIndex={actionTypesIndex}
+                connectors={connectors}
+                messageVariables={messageVariables}
+                summaryMessageVariables={summaryMessageVariables}
+                defaultActionMessage={defaultActionMessage}
+                actionTypeRegistry={actionTypeRegistry}
+                onDeleteAction={() => {
```

Comment:
I think `onDeleteConnector` on L423 is doing the same thing, we should create just one callback for that if possible